storage/innobase/include/row0mysql.h

```diff
@@ -686,6 +686,8 @@ struct row_prebuilt_t {
 	/** The MySQL table object */
 	TABLE*		m_mysql_table;
 
+    bool full_table_lock=false;
```

Comment:
The declaration of an initializer expression would introduce an implicit constructor `row_prebuilt_t::row_prebuilt_t()`. That in turn could lead into trouble with the GCC 6 `-flifetime-dse`, because the rest of the fields would not be zero-initialized.

All fields will be zero-initialized in `row_create_prebuilt()` by invoking `mem_heap_zalloc()`.

The field name is misleading: it should be `full_table_scan`. It is also missing a preceding Doxygen comment that explains when the flag would be set and what its impact is.

Reply:
Done.